eng/dockerfile-templates/monitor/Dockerfile

```
@@ -0,0 +1,70 @@
+{{
+    _ .NET major version matches the major version of dotnet-monitor ^
+    set dotnetMajor to split(PRODUCT_VERSION, ".")[0] ^
+    set dotnetMajorMinor to cat(dotnetMajor, ".0") ^
+    set targetFrameworkMoniker to cat("net", dotnetMajorMinor) ^
+    set targetFrameworkMonikerRegex to cat("net", dotnetMajor, "[.]0") ^
+    set installerBaseTag to cat(VARIABLES[cat("sdk|", dotnetMajorMinor, "|product-version")], "-", OS_VERSION, ARCH_TAG_SUFFIX) ^
+    set monitorMajorMinor to join(slice(split(PRODUCT_VERSION, "."), 0, 2), ".") ^
+    set monitorVersionVariable to when(monitorMajorMinor != "7.0", "product-version", "build-version") ^
+    set monitorVersion to VARIABLES[cat("monitor|", monitorMajorMinor, "|", monitorVersionVariable)] ^
+    set buildVersion to VARIABLES[cat("monitor|", monitorMajorMinor, "|build-version")] ^
+    set monitorSha to VARIABLES[cat("monitor|", monitorMajorMinor, "|sha")] ^
+    set imageBaseTag to cat(VARIABLES[cat("dotnet|", dotnetMajorMinor, "|product-version")], "-", OS_VERSION, ARCH_TAG_SUFFIX) ^
+    _ All .NET Monitor versions publish to the same storage account, thus use the latest storage account scheme. ^
+    _ However, the latest 6.0 release still uses dotnetcli storage account. ^
```

You can have multiple lines in one comment like the following:

```suggestion
    _ All .NET Monitor versions publish to the same storage account, thus use the latest storage account scheme.
      However, the latest 6.0 release still uses dotnetcli storage account. ^
```